_pages/visas.md

```diff
@@ -1,29 +1,24 @@
 ---
-description: Information about obtaining a Visa to visit DjangoCon US
-heading: Visa Support at DjangoCon US
+description: Information about obtaining a Visa to visit DjangoCon Europe
+heading: Visa Information
 layout: default
 permalink: /visas/
-published: false
-title: Visa Support at DjangoCon US
+published: true
+title: Visa Information
 ---
 
-## Do I need a visa to enter the USA?
+## Do I need a visa to attend DjangoCon Europe?
 
-That depends on what country issued your passport; see the [US State Department](https://travel.state.gov/content/travel/en/us-visas/tourism-visit.html) website for more information.
+If you’re planning to attend DjangoCon Europe, first check whether you need a visa to 
+visit the UK. Fill in the form on [this page](https://www.gov.uk/check-uk-visa/y) to check 
+whether you need a visa.  A standard UK visitor visa costs £100, and may take around 6 weeks to process.  You can find more information on visa requirements for visiting the UK 
+here.
```

Comment:
As a general rule we should always use a link label that’s descriptive of what the link takes the user to, so screen reader users can more easily navigate the page link-by-link.

We should probably link to the parent page (`https://www.gov.uk/check-uk-visa`), so people understand where in the site they are and can see related resources.

```suggestion
visit the UK. Fill in the official [Check if you need a UK visa](https://www.gov.uk/check-uk-visa) form to check. 
A standard UK visitor visa costs £100, and may take around 6 weeks to process. You can find more information on visa requirements for visiting the UK here.
```

Is the last "here" missing a link?